public/app/features/visualization/data-hover/DataHoverView.tsx

```diff
@@ -110,6 +113,10 @@ export const DataHoverView = ({ data, rowIndex, columnIndex, sortOrder, mode, he
 
   const { displayValues, links } = dispValuesAndLinks;
 
+  if (config.featureToggles.newVizTooltips) {
+    return <ExemplarHoverView displayValues={displayValues} links={links} header={header} />;
+  }
```

Comment:
I think we use DataHoverView in a few other places (outside of the exemplar use case) 🤔

See this example with BarChart
<img width="535" alt="needs to be exemplar specific" src="https://github.com/grafana/grafana/assets/22381771/93ac1f15-5074-4a3b-8c88-013b1b0a04f9">


I think given this we might want to have some way of determining whether or not we are displaying an exemplar - maybe for time being if the passed header prop = `Exemplar` or something (not ideal but 🤷🏻‍♂️)

Reply:
Totally missed that. I added a check for the header.